contracts/LimitsChecker.sol

```diff
@@ -223,6 +227,22 @@ contract LimitsChecker is AccessControl {
         return _getCurrentPeriodState(limit, spentAmount, currentPeriodEndTimestamp);
     }
 
+    /// @notice Sets address of BokkyPooBahsDateTime contract
+    /// @dev Need this to be able to replace the contract in case of a bug in it
+    /// @param _bokkyPooBahsDateTimeContract New address of the BokkyPooBahsDateTime library
+    function setBokkyPooBahsDateTimeContract(address _bokkyPooBahsDateTimeContract)
+        external
+        onlyRole(SET_LIMIT_PARAMETERS_ROLE)
+    {
+        require(
```

Comment:
If went to actual impl — let's do exact check upon DAO Agent address or default admin role if we have one here

Reply:
Let's rename the role to `SET_LIMIT_PARAMETERS_ROLE` and be it

Reply:
Renamed